dpc/src/program/public_variables.rs

```diff
@@ -15,41 +15,30 @@
 // along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
 
 use crate::Parameters;
-use snarkvm_algorithms::{merkle_tree::MerklePath, SNARK};
 use snarkvm_fields::{ConstraintFieldError, ToConstraintField};
 
-/// Program index, path, verifying key, and proof.
-#[derive(Derivative)]
-#[derivative(Clone(bound = "C: Parameters"))]
-pub struct Execution<C: Parameters> {
-    pub circuit_index: u8,
-    pub program_path: MerklePath<C::ProgramIDTreeParameters>,
-    pub verifying_key: <C::ProgramSNARK as SNARK>::VerifyingKey,
-    pub proof: <C::ProgramSNARK as SNARK>::Proof,
-}
-
 #[derive(Derivative)]
 #[derivative(
     Clone(bound = "C: Parameters"),
     Debug(bound = "C: Parameters"),
     Default(bound = "C: Parameters")
 )]
-pub struct ProgramPublicVariables<C: Parameters> {
-    pub local_data_root: C::LocalDataRoot,
+pub struct PublicVariables<C: Parameters> {
     pub record_position: u8,
```

Comment:
Just to confirm: The PR said that we removed the record position from the `Record` struct, but `position` will still be used in other places to reference to this record right?

Reply:
Correct, there is a discrepancy that we need to resolve (via a standard convention) for the `leaf_index` and `record_position` as both are part of local data. They arose for different reasons, but are ordered in the same manner.